tensorflow/core/debug/debug_grpc_io_utils_test.cc

```diff
@@ -25,6 +25,9 @@ limitations under the License.
 #include "tensorflow/core/platform/env.h"
 #include "tensorflow/core/util/event.pb.h"
 
+// TODO(cais): Support grpc:// debug URLs in open source once Python grpc
+//   genrule becomes available. See b/23796275.
+#if defined(PLATFORM_GOOGLE)
```

Comment:
There's no `#define` for CMake builds, but I don't see why you'd need one.

Excluding this test in `tf_tests.cmake` (e.g. by adding its filename as another line [here](https://github.com/tensorflow/tensorflow/blob/43c71a03380d8de18202cc399563814b2f438cd2/tensorflow/contrib/cmake/tf_tests.cmake#L241)) would prevent it being run in the CMake on *any* platform. (Are you instead concerned about the Linux/Mac *Bazel* tests? I guess these would start failing without this `#if`....)

I still think it would be cleaner to exclude the test explicitly in the CMake/Bazel build files, rather than building a test that "passes" but doesn't do anything.